### ISSUE TITLE
LevOCR on custom datasets

### ISSUE THREAD

**Issue by ngocnd2402** [AlibabaResearch/AdvancedLiterateMachinery#44]
Can i finetune LevOCR on my custom dataset for my language (Vietnamese) ?

**Comment by wdp-007** [comment]
Of course, it is possible. One thing to note is that you will need to gather data in your language and retrain the language model of LevOCR.
